airbyte-db/db-lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -1,209 +1,205 @@
-// The content of the file is just to have a basic idea of the current state of the database and is not fully accurate. 
-// It is also not used by any piece of code to generate anything. 
-// It doesn't contain the enums created in the database and the default values might also be buggy. 
+// The content of the file is just to have a basic idea of the current state of the database and is not fully accurate.
+// It is also not used by any piece of code to generate anything.
+// It doesn't contain the enums created in the database and the default values might also be buggy.
 
-create table "public"."actor"(
+create table "public"."actor" (
   "id" uuid not null,
   "workspace_id" uuid not null,
   "actor_definition_id" uuid not null,
   "name" varchar(256) not null,
   "configuration" jsonb not null,
-  "actor_type" actor_type not null,
-  "tombstone" bool not null default false,
-  "created_at" timestamptz(35) not null default null,
-  "updated_at" timestamptz(35) not null default null,
+  "actor_type" "public"."actor_type" not null,
+  "tombstone" boolean not null default false,
+  "created_at" timestamp(6) with time zone not null default current_timestamp,
```

Comment:
Is this type change expected? It feels more than just cosmetic.

Reply:
Unsure.  We are two years behind in terms of jOOQ releases, so its not surprising that they changed the datatypes oer that period.

Reply:
could this have an impact on the new DB? https://medium.com/building-the-system/how-to-store-dates-and-times-in-postgresql-269bda8d6403#:~:text=timestamp%20vs.-,timestamptz,%E2%80%9Ctimestamp%E2%80%9D%20doesn't.&text=From%20the%20previous%20example%3A,')%20from%20the%20original%20value. This is what I found to explain de difference between timestampz and timestamp. It feel that we are changing the type of the column and I'm not sure about what could be the consequences on this

Reply:
To be clear, this is just changing in the those dump files, which are made from a new containerized database migrated from scratch.  The question is whether or not the generated jOOQ classes will be compatible with the existing database.  We won't know for sure until we test this in Cloud in one of the dev environments.

Reply:
@gosusnp @benmoriceau I looked into this some more and I don't believe that the timestampz -> timestamp with time zone is an issue.  The generated jOOQ model code still uses the same datatype.  This really just impacts how jOOQ interprets the schema when exporting it to a file.  The type in the database is still the same as before, as it utilizes Flyway to actually create/migrate the database.  I suppose the issue here is only for those who attempt to re-create the database from the dumped schema file instead of using the Bootloader to manage the creation and migration of the database.  Therefore, I don't think that there is much risk here.

Reply:
Thanks, sounds like as long as we make sure to have a test with a rolling upgrade, we should be good then.